app/controllers/api/v2/table_preferences_controller.rb

```diff
@@ -1,26 +1,29 @@
 module Api
   module V2
     class TablePreferencesController < V2::BaseController
-      include Api::Version2
```

Comment:
Why?

Reply:
Actually, why we include this everywhere since we have it already included in `V2::BaseController`?...